src/main/java/com/openshift/internal/restclient/URLBuilder.java

```diff
@@ -42,7 +41,7 @@
 	private String baseUrl;
 	private String kind;
 	private String name;
-	private Map<String, String> params = new HashMap<String, String>();
+	private ArrayList<AbstractMap.SimpleEntry<String,String>> params = new ArrayList<>();
 	private final IApiTypeMapper typeMappings;
```

Comment:
Yep:
$ cat /tmp/file
would translate into the URL:
....exec?command=cat&command=/tmp/file....